queries/models/br_rj_riodejaneiro_brt_gps/schema.yml

```
@@ -64,32 +64,33 @@ models:
+  #     - name: id_migracao_trajeto
+  #       description: ""
+  #     - name: sentido
+  #       description: "{{ doc('sentido') }}"
+  #     - name: trajeto
+  #       description: "{{ doc('trajeto') }}"
+  #     - name: timestamp_captura
+  #       description: "{{ doc('timestamp_captura') }}"
+  #     - name: data
+  #       description: "{{ doc('data_captura') }}"
+  #     - name: hora
+  #       description: "Hora do 'timestamp_captura'"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)